src/lib/config.ts

```diff
@@ -59,6 +59,10 @@ export function intVar<R>(varName: string, defaultValue?: R): number | R {
 }
 
 export const API_HOST = requiredVar('API_HOST');
+export const API_HEARTBEAT_STATE_TIMEOUT_SECONDS = intVar(
```

Comment:
This is the grace period; the time allowed from when it has missed it's state poll checkin and when we mark it as offline in the DB. 15 secs default value.